src/http/index.js

```diff
@@ -92,8 +92,13 @@ function HttpApi (repo, config, cliArgs) {
           }
 
           // CORS is enabled by default
+          // TODO: shouldn't, fix this
           this.server = new Hapi.Server({
-            connections: { routes: { cors: true } }
+            connections: {
+              routes: {
+                cors: true
```

Comment:
Seems we're setting `cors` to `true` here and also adding headers ourselves in `src/http/gateway/resources/gateway.js`, this configuration should be enough.